src/compiler/transform/context.ts

```diff
@@ -14,7 +14,7 @@ import { FunctionDeclaration } from '../ast/statements/function-declaration';
 /**
  * An interface for transformer context
  */
-export interface Context {}
+export interface Context {} // eslint-disable-line @typescript-eslint/no-empty-interface -- fix later(TODO)
```

Comment:
is the type really needed?

Reply:
Still WIP that code so you can put `//` in front of it